### DISCUSSION TITLE
Activate a Function when the checkmark appears

### DISCUSSION THREAD

**Discussion by rbalonek** [add2cal/add-to-calendar-button#333] [Q&A] [answered]
I know there is a hideCheckmark prop, but I was trying to activate a function when the checkmark appears:

`
 function isVisibleByClassName(className) {
    const element = document.getElementsByClassName(className)[0];
    if (!element) return false;
    return element.offsetWidth > 0 && element.offsetHeight > 0;
  }

  console.log(isVisibleByClassName("atcb-checkmark"))
`

Is there a current way to do that or is it something that can be implemented?

**Comment by jekuer** [comment, marked as answer]
Hi,
You could observe the atcb-last-event attribute at the host element (`<add-to-calendar-button>`). When it turns to "success:buttonID", that's when we show the checkmark.
"buttonID" will be the generated ID of the button. Usually atcb-btn-1.
